hor:
  sensing:
    xmin: -10
    xmax: -1
    ymin: -10
    ymax: 10

  line:
    theta_min: 140 # specify lower to higher	w.r.t laser frame sim (wall in front): 45 to 135 real (wall to left): 135 to 225
    theta_max: 220
    resolution: 0.1 # currently hardcoded in code
    threshold: 10

    min_line_separation_dist: 2 # (metres)
    min_line_separation_angle: 45 # (degrees)

vert:
  sensing:
    xmin: -2 # X-axis is the interesting vertical axis (Z) in the real world
    xmax: 4 
    ymin: 1 # Y-axis is the distance to the bridge (to the left)
    ymax: 10
  line:

    theta_min: 60 # specify specify lower to higher	w.r.t laser frame	
    theta_max: 120
    resolution: 0.3 
    threshold: 7

    min_line_separation_dist: 2 #  (metres)
    min_line_separation_angle: 45 #  (degrees)

control:
  laser_rf_offset: 0 # horizontal lidar leads robot x axis (east) by 0 degrees in simulation and 0 degrees on the actual hex

  max_vel_h: 0.5 # max hold (compensation) velocity in hor direction
  max_vel_z: 0.4 # max compensation velocity in Z direction
  nominal_vel: 0 # nominal velocity independent 	of compensation velocity

  Kp: 0.4
  Kd: 0.2
  Ki: 0
  Z_Kp: 0.4
  Z_Kd: 0.2
  Z_Ki: 0

flight:
  desired_wall_dist: 4.5
  desired_buffer: 2.5         

  move_threshold_vertical: 0.7
  move_threshold_horizontal: 0.7





